### ISSUE TITLE
Issue Playing large MKV files (+30GB + DTS HDMA Audio)

### ISSUE THREAD

**Issue by toro1977** [koying/SPMC#229]
1) Playing Large MKV Files (+30GB) I'm unable to seek, it just gets frozen
2) I cannot see my external HDD from SPMC, I have to play movies from the File Explorer.

My setup:
Nvidia TV 16 GB
Micro SD 32 GB as internal Storage.
MY passport 1TB (Where I play movies from)
7.1 Setup DTS HDMA enabled 

If anybody went trough this issue sometime please let me know.

thanks a lot!!

**Comment by ghost** [comment]
wifi?

**Comment by toro1977** [comment]
Yes, wifi, but the streaming is not the problem, all the files i'm talking about are local ones.In fact using KODI / Exodus, streamig goes perfect and seeking as well.

Thanks.

**Comment by toro1977** [comment]
As I could test last night, it happens when I come back from Sleep mode, but when i come back from shut down mode, it runs perfect.

Weird!

**Comment by koying** [comment]
Confusing.
Are you saying that after a Shield reboot, you can see your HDD and seek the large MKVs on it?

**Comment by toro1977** [comment]
Yes, when i go to sleep mode and come back it seems to be the issue.When I shut down (No sleep mode) it works.i know it's strange but it is the way it works so far for me, i'll keep on testing tonight.

thanks.

another one taking advantage "you are here". how can i play BDMV folders other than fro Stream Folder? I mean because of the subtitles for example.

Thanks a lot.

**Comment by koying** [comment]
Work fine here (50Gb from Passport)

**Comment by toro1977** [comment]
hi there,

There are times when SPMC detects my HDD and there are times when it doesnt. in the case it doesnt, ES file Explorer does, and i'm able to play movies in SPMC opening them from the explorer, when so, I can reproduce the issue, but the times when I open the movies directly from SPMC, it works perfect.

Could it be my HDD, the file Explorer or SPMC? or other cause?

Thanks a lot!!
